bower.json

```diff
@@ -26,7 +26,8 @@
     "short-code-forms": "jywarren/short-code-forms#~0.0.1",
     "leaflet-blurred-location": "publiclab/leaflet-blurred-location#master",
     "chart.js": "v2.7.0",
-    "typeahead.js-browserify": "Javier-Rotelli/typeahead.js-browserify#~1.0.7"
+    "typeahead.js-browserify": "Javier-Rotelli/typeahead.js-browserify#~1.0.7",
+    "noty": "3.1.4"
```

Comment:
Can you also add this to the Yarn PR so we don't break it in the future -- in #2659 ? Thanks!